addon/components/o-s-s/password-input.ts

```diff
@@ -130,7 +130,10 @@ export default class OSSPasswordInput extends Component<OSSPasswordInputArgs> {
   }
 
   private testAllValidators(): boolean {
-    return this.inputValidators.every((index: InputValidator) => this.validatorSet[index].regex.test(this.args.value!));
+    return this.inputValidators.every((key: InputValidator) => {
+      if (!this.validatorSet[key]) return false;
```

Comment:
just for extra safety, right ?

Reply:
yep! but i got an error about it the first time i did the migration and now it works without this change 😩